src/testCommon/java/uk/gov/hmcts/darts/test/common/data/builder/CustomCourtCaseEntity.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.hmcts.darts.test.common.data.builder;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.beanutils.BeanUtils;
+import org.hibernate.AssertionFailure;
+import uk.gov.hmcts.darts.common.entity.CaseRetentionEntity;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
+import uk.gov.hmcts.darts.common.entity.CourthouseEntity;
+import uk.gov.hmcts.darts.common.entity.DefenceEntity;
+import uk.gov.hmcts.darts.common.entity.DefendantEntity;
+import uk.gov.hmcts.darts.common.entity.EventHandlerEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.JudgeEntity;
+import uk.gov.hmcts.darts.common.entity.MediaLinkedCaseEntity;
+import uk.gov.hmcts.darts.common.entity.ProsecutorEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.retention.enums.RetentionConfidenceReasonEnum;
+import uk.gov.hmcts.darts.retention.enums.RetentionConfidenceScoreEnum;
+
+import java.lang.reflect.InvocationTargetException;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+public class CustomCourtCaseEntity extends CourtCaseEntity implements DbInsertable<CourtCaseEntity> {
```

Comment:
A more descriptive name would be better such as TestCourtCaseEntity as custom does not indicate what it is for (same for all with this prefix)